src/js/utils/apiClient.jsx

```diff
@@ -54,19 +54,7 @@ const handleSuccess = response => response;
 const handleError = (error) => {
   switch (error.response.status) {
     case 400:
-      Alert.error(`<div>Bad request.</br>${_.get(error, 'response.data.errorMessage', '')}</div></br>
```

Comment:
Are you sure that there is no other error with status 400 returned, that will need this version of handling?

Reply:
Not right now. Only validation exception is currently returning code 400.